addons/calendar/models/calendar_event.py

```diff
@@ -349,11 +350,52 @@ def _inverse_dates(self):
                     'stop': enddate.replace(tzinfo=None)
                 })
 
-    def _compute_attendee(self):
-        mapped_attendees = self._find_attendee_batch()
-        for meeting in self:
-            attendee = mapped_attendees[meeting.id]
-            meeting.attendee_status = attendee.state if attendee else 'needsAction'
+    @api.model
+    def get_compatible_attendee_status(self, domain):
+        """
+        Return the list of status corresponding to the current domain
+        :param domain: A list of leaf domain searching the attendee_status field.
+        :return: a list of accepted domains
+        :rtype: list
+        """
+        if not domain:
+            domain = []
+        attendee_status = []
+        for domain_leaf in domain:
+            attendee_status += self.get_attendee_status(domain_leaf[1], domain_leaf[2])
+        return attendee_status or list(ATTENDEE_STATUS_VALUES)
+
+    @api.model
+    def get_attendee_status(self, operator, value):
```

Comment:
I don't think this method should be public. :thinking:

Reply:
Fixed!